src/simulations/predictions/prediction_utils.jl

```diff
@@ -29,7 +29,7 @@ function batch_iceflow_PDE!(glacier_idx::I, simulation::Prediction) where {I <:
     glacier = simulation.glaciers[glacier_idx]
 
     glacier_id = isnothing(glacier.rgi_id) ? "unnamed" : glacier.rgi_id
-    println("Processing glacier: ", glacier_id)
+    @info "Processing glacier $(glacier_id) for PDE forward simulation"
```

Comment:
I'm not super fond of this use of `@info`. I would really restrict it to only specific messages, not things that are repeated multiple times for many glaciers. Seeing so many info logs all at once defeats its purpose.